sample/src/main/scala/za/co/absa/spline/sample/batch/CodelessInitSampleJob.scala

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2019 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.spline.sample.batch
+
+import org.apache.spark.sql.SaveMode
+import za.co.absa.spline.sample.SparkApp
+
+/**
+  * A sample job where no explicit Spline code was used to initialize Spline. Only Spark configuration was used.
+  */
+object CodelessInitSampleJob extends SparkApp(
```

Comment:
IMHO we should have at least one simple end to end test that will be run in unit test.
It should use just make Spark context, do some computation and check if computation was recorded.
Even smoke test would be sufficient.

Reply:
I wanted add some, but on develop branch only to avoid too much conflicts.